pkg/apis/resource/extension_view.go

```diff
@@ -530,15 +533,10 @@ func (r *CollectionRouteUpgradeRequest) Validate() error {
 		if r.ReuseAttributes {
 			input.Attributes = entity.Attributes
 			input.ComputedAttributes = entity.ComputedAttributes
-		} else {
-			en := updateInputsItemToResource(entity.Type, input, r.Project, r.Environment)
-
-			input.ComputedAttributes, err = pkgresource.GenComputedAttributes(r.Context, r.Client, en)
-			if err != nil {
-				return err
-			}
 		}
 
+		en := updateInputsItemToResource(entity.Type, input, r.Project, r.Environment)
```

Comment:
We don't call this if not ReuseAttributes before. Is this expected change?

Reply:
This variable Is only used in computed resources, the func will new `*model.Resource`, the input data will not be effected.